boefjes/boefjes/plugins/kat_nikto/schema.json

```diff
@@ -0,0 +1,32 @@
+{
+  "title": "Arguments",
+  "type": "object",
+  "properties": {
+    "PROXYHOST": {
+      "title": "PROXYHOST",
+      "maxLength": 256,
+      "type": "string"
+    },
+    "PROXYPORT": {
+      "title": "PROXYPORT",
+      "maxLength": 256,
+      "type": "string"
+    },
+    "PROXYUSER": {
+      "title": "PROXYUSER",
+      "maxLength": 256,
+      "type": "string"
+    },
+    "PROXYPASS": {
+      "title": "PROXYPASS",
+      "maxLength": 256,
+      "type": "string"
+    },
+    "USERAGENT": {
+      "title": "USERAGENT",
+      "maxLength": 256,
+      "type": "string"
+    }
```

Comment:
Perhaps you could use the `HTTP_PROXY` (and optionally `HTTPS_PROXY`) for this? This is a common proxy URI configuration. This will save a few variables and make the form less complex (given that it has a good example, e.g. `http://user:pass@1.2.3.4:3128/`

Reply:
These are config variables required for Nikto. If we would make the boefje's schema less complex, then it would move the complexity to the code of the boefje. 

I believe since the user chooses to activate the Nikto boefje, they would know that Nikto requires these variables which would make them self-explaining.

Reply:
It's not complex code. You can use `urlparse` from `urllib.parse` and simply pass on the parsed components (hostname, port, user and password) to Nikto.

The reason I'm bringing this up is to reduce overall complexity and work towards a standard approach. A lot of Boefjes we make usually contain some kind of (HTTP) web request and usually use tools or dependencies that work with these `HTTP_PROXY` and `HTTPS_PROXY` variables convention. If we'd have to specify these multiple variables just to specify proxy options for every other proxy-enabled Boefje, that would be quite cumbersome.

What do you think @underdarknl?

Reply:
My thought when developing this boefje was that users that have experience with Nikto would use this boefje. But I realize now that is not the case. It is OpenKAT users that (might never have used Nikto) will use this boefje.

I will implement it in the way you have suggested, thank you 😅

Reply:
We should still put the globally available HTTP_PROXY var in the boefje's schema for it to bleed trough into the boefjes runtime env. Beyond that we could easily transpose whatever value is in there to the vars that NIKTO requires.

Reply:
Is HTTP_PROXY something that OpenKAT contains? I have not heard of that before. I can also not find anything about it in the repo

Reply:
I understand the comment correctly now, I assumed `HTTP_PROXY` was something provided by OpenKAT.

e6cf370 fixes this.